core/modules/simpletest/backdrop_web_test_case.php

```diff
@@ -825,10 +835,17 @@ function __construct($test_id = NULL) {
    * method.
    */
   protected function setUp() {
-    global $conf;
+    global $conf, $language;
 
     // Store necessary current values before switching to the test environment.
     $this->originalFileDirectory = config_get('system.core', 'file_public_path');
+    $this->verboseDirectoryUrl = file_create_url($this->originalFileDirectory . '/simpletest/verbose');
+
+    // Set up English language.
+    $this->originalLanguage = $language;
+    $this->originalLanguageDefault = variable_get('language_default');
+    unset($conf['language_default']);
+    $language = language_default();
```

Comment:
I'm not sure if this commit is needed at all in Backdrop. We don't store the language default values in `$conf` at all (though we do have a static variable cache). I think it's the use of globals that throws off D7, that may not be an issue for us.